components/molecules/TranslationsWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { SupportedLocale } from '../../@types';
+
+type Props = {
+  locale: SupportedLocale
+  children: JSX.Element
+};
+
+export const LocaleContext = React.createContext<{ locale: SupportedLocale }>({ locale: 'en' });
```

Comment:
Change this

```suggestion
export const TranslationContext = React.createContext<{ locale: SupportedLocale }>({ locale: 'en' });
```

Reply:
Also, maybe this shouldn't be a molecule - we could make a directory under `components/` like `top_level_wrappers` (for wrapping pages)?